src/app/dashboard/Missions/CreateMission.js

```diff
@@ -0,0 +1,482 @@
+                      type="number"
+                      variant="outlined"
+                      InputProps={{ inputProps: { min: 0 } }}
+                      className={classes.numSelector}
+                      value={amount}
+                    />
+                  </div>
+                );
+              })}
+              <Button className={classes.selectCreateButton} onClick={(e) => addToFBCount(e)}>
+                <AddCircleIcon style={{ marginRight: "5px", color: "#3739B5" }} />
+                Add Food Box
+              </Button>
+            </FormControl>
+          </>
+        ) : null}
+        <Divider variant="middle" />
+        <div className={classes.optionalTagBox}>
+          <Typography className={classes.inputHeader}>Funding Details </Typography>
+          <Typography className={classes.optionalTag}> - optional</Typography>
+        </div>
+        <FormControl className={classes.form}>
+          <Select
+            variant="outlined"
+            className={classes.select}
+            value={missionFund}
+            onChange={handleChangeFund}
+            startAdornment={
+              <InputAdornment position="start">
+                <AttachMoneyIcon style={{ color: "#3739B5" }} />
+              </InputAdornment>
+            }
+          >
+            {/* THIS WILL HAVE TO HAVE A DYNAMIC INITIAL VALUE */}
+            <MenuItem className={classes.menuItem} value="Not yet funded">
+              Not yet funded
+            </MenuItem>
+            <MenuItem className={classes.menuItem} value="Funded">
+              Funded
+            </MenuItem>
+          </Select>
+        </FormControl>
+        <Divider variant="middle" />
+        <div className={classes.dualForm}>
+          <FormControl className={classes.form2}>
+            <div className={classes.optionalTagBox}>
+              <Typography className={classes.inputHeader2}>Recipient Details </Typography>
+              <PeopleIcon className={classes.header2Icons} />
+            </div>
+            <Select
+              className={classes.dualSelect}
+              labelid="name"
+              placeholder="Name"
+              value={recipient.name}
+              variant="outlined"
+              name="name"
+              onChange={handleRecipient}
+            >
+              <MenuItem className={classes.menuItem} value="default">
+                Select Recipient
+              </MenuItem>
+            </Select>
+            <br />
+            <TextField
+              variant="outlined"
+              name="phoneNumber"
+              onChange={handleRecipient}
+              className={classes.dualText}
+              value={recipient.phoneNumber}
+              labelid="phone"
+              placeholder="(123) 456-7890"
+              InputProps={{
+                startAdornment: (
+                  <InputAdornment position="start">
+                    <PhoneIcon style={{ color: "#3739B5" }} />
```

Comment:
@thirdeyeclub  Can you move this inline style out into the styles object? and we should be referring to theme colours. Hardcoding styles here one-off will make it difficult to maintain.